app/locales/fr/messages.po

```diff
@@ -120,59 +120,67 @@ msgid "button.share"
 msgstr "Partager"
 
 #: app/configurator/components/ui-helpers.ts:562
-#: app/configurator/map/map-chart-options.tsx:166
+#: app/configurator/map/map-chart-options.tsx:204
 msgid "chart.map.layers.area"
 msgstr "Zones"
 
-#: app/configurator/map/map-chart-options.tsx:250
+#: app/configurator/map/map-chart-options.tsx:288
 msgid "chart.map.layers.area.discretization.continuous"
 msgstr "Continue"
 
-#: app/configurator/map/map-chart-options.tsx:263
+#: app/configurator/map/map-chart-options.tsx:301
 msgid "chart.map.layers.area.discretization.discrete"
 msgstr "Discrète"
 
-#: app/configurator/map/map-chart-options.tsx:311
+#: app/configurator/map/map-chart-options.tsx:344
 msgid "chart.map.layers.area.discretization.jenks"
 msgstr "Jenks (intervalles naturels)"
 
-#: app/configurator/map/map-chart-options.tsx:304
+#: app/configurator/map/map-chart-options.tsx:337
 msgid "chart.map.layers.area.discretization.quantiles"
 msgstr "Quantiles (distribution homogène des valeurs)"
 
-#: app/configurator/map/map-chart-options.tsx:297
+#: app/configurator/map/map-chart-options.tsx:330
 msgid "chart.map.layers.area.discretization.quantize"
 msgstr "Intervalles égaux"
 
-#: app/configurator/components/chart-configurator.tsx:672
+#: app/configurator/components/chart-configurator.tsx:680
 #: app/configurator/components/ui-helpers.ts:558
-#: app/configurator/map/map-chart-options.tsx:64
+#: app/configurator/map/map-chart-options.tsx:96
 msgid "chart.map.layers.base"
-msgstr "Couche de base"
+msgstr "Affichage de la carte"
 
-#: app/configurator/map/map-chart-options.tsx:68
+#: app/configurator/map/map-chart-options.tsx:100
 msgid "chart.map.layers.base.show"
-msgstr "Afficher"
+msgstr "Afficher la carte du monde"
 
-#: app/configurator/map/map-chart-options.tsx:170
-#: app/configurator/map/map-chart-options.tsx:394
+#: app/configurator/map/map-chart-options.tsx:109
+msgid "chart.map.layers.base.view.dynamic"
+msgstr "Vue dynamique"
+
+#: app/configurator/map/map-chart-options.tsx:118
+msgid "chart.map.layers.base.view.locked"
+msgstr "Vue verrouillée"
```

Comment:
The translation here is not immediately understandable in French. While trying to find a good other translation, I wondered if in fact we should not have "Locked map" with a [Switch](https://mui.com/material-ui/react-switch/#main-content). This way we do not have to find another word for the "dynamic" here (I find dynamic even in English a bit vague in this case). Since we have only two states, the Switch seems warranted.

Reply:
I'd use "Carte verrouillée" in French.